sig-and-wg/sig-core/proposals/event-bus-monitoring-metrics.md

```diff
@@ -0,0 +1,113 @@
+# Event Bus Monitoring Metrics Proposal
+
```

Comment:
Shouldn't we consider metrics to figure out the mismatch between Kyma subscriptions and whatever backing resources required to ensure event delivery?
There could be other scenarios where things can go wrong and Kyma subscriptions created might not be functional. Please consider all those scenarios and propose metrics for them.